src/Domain/CoordinateMaps/TimeDependent/Skew.hpp

```diff
@@ -0,0 +1,246 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+#include <memory>
+#include <optional>
+#include <string>
+#include <unordered_map>
+#include <unordered_set>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Domain/FunctionsOfTime/FunctionOfTime.hpp"
+#include "Utilities/TypeTraits/RemoveReferenceWrapper.hpp"
+
+/// \cond
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace domain::CoordinateMaps::TimeDependent {
+/*!
+ * \ingroup CoordMapsTimeDependentGroup
+ * \brief %Time dependent coordinate map that keeps the $y$ and $z$ coordinates
+ * unchanged, but will distort (or skew) the $x$ coordinate based on functions
+ * of time and radial distance to the origin.
+ *
+ * \details This coordinate map is only available in 3 dimensions and assumes
+ * all coordinates are given with respect to the origin of the coordinate
+ * system.
```

Comment:
I don't understand what this means. What else would I give the coordinates relative to?

Reply:
This was to try and disambiguate the origin from the center of the skew map, but this does make it confusing. I can remove it when I squash.

Reply:
Ah, okay. That's what I assumed but was only able to figure it out because I reviewed Mark's Spec PR :) Thanks for removing it!